### ISSUE TITLE
homebridge-hue v.0.14.0 removes non-hue lights from HomeKit

### ISSUE THREAD

**Issue by anarqist** [ebaauw/homebridge-hue#1191]
#### Issue
After the latest update of Homebridge-Hue it removed my OSRAM lightstrips connected to my hue bridge from HomeKit. Homebridge-Hue v0.13.70 works fine and a downgrade to that version brought back my lightstrips into HomeKit. 

#### Log Messages

```
[37m[10/22/2024, 12:20:52 AM] [39m[36m[Hue][39m Loaded homebridge-hue v0.14.0 child bridge successfully
[37m[10/22/2024, 12:20:52 AM] [39mLoaded 0 cached accessories from cachedAccessories.0E4162261194.
[37m[10/22/2024, 12:20:52 AM] [39m[36m[Hue][39m homebridge-hue v0.14.0, node v20.18.0, homebridge v1.8.4
[37m[10/22/2024, 12:20:52 AM] [39m[36m[Hue][39m [33mconfig.json: effects: invalid key[39m
[37m[10/22/2024, 12:20:52 AM] [39m[36m[Hue][39m Hue Bridge: Signify Netherlands B.V. BSB002 bridge v1967054010, api v1.67.0
[37m[10/22/2024, 12:20:52 AM] [39m[36m[Hue][39m [31mHue Bridge: error: /lights/15: {"state":{"on":true,"bri":76,"hue":768,"sat":251,"effect":"none","xy":[0.672,0.31],"ct":370,"alert":"select","colormode":"xy","mode":"homeautomation","reachable":true},"swupdate":{"state":"notupdatable","lastinstall":"1980-01-06T03:13:02"},"type":"Extended color light","name":"Lowboard","modelid":"Flex RGBW","manufacturername":"OSRAM","productname":"Extended color light","capabilities":{"certified":false,"control":{"colorgamuttype":"other","ct":{"min":151,"max":370}},"streaming":{"renderer":false,"proxy":false}},"config":{"archetype":"classicbulb","function":"mixed","direction":"omnidirectional"},"uniqueid":"84:18:26:00:00:da:a8:c7-03","swversion":"V1.03.07"}[39m
[31mTypeError: Cannot read properties of undefined (reading 'swversion')[39m
[31m    at Object.fix (file:///usr/local/lib/node_modules/homebridge-hue/lib/HueLight.js:186:20)[39m
[31m    at HueLight.setConfig (file:///usr/local/lib/node_modules/homebridge-hue/lib/HueLight.js:512:20)[39m
[31m    at new HueLight (file:///usr/local/lib/node_modules/homebridge-hue/lib/HueLight.js:302:10)[39m
[31m    at HueAccessory.exposeLightResource (file:///usr/local/lib/node_modules/homebridge-hue/lib/HueAccessory.js:223:21)[39m
[31m    at HueAccessory.exposeLightResources (file:///usr/local/lib/node_modules/homebridge-hue/lib/HueAccessory.js:213:14)[39m
[31m    at HueAccessory.expose (file:///usr/local/lib/node_modules/homebridge-hue/lib/HueAccessory.js:103:10)[39m
[31m    at HueBridge.exposeResources (file:///usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:571:51)[39m
[31m    at HueBridge.accessories (file:///usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:101:18)[39m
[31m    at processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async Promise.all (index 0)[39m
[37m[10/22/2024, 12:20:52 AM] [39m[36m[Hue][39m [31mHue Bridge: error: /lights/16: {"state":{"on":true,"bri":76,"hue":768,"sat":251,"effect":"none","xy":[0.672,0.31],"ct":370,"alert":"select","colormode":"xy","mode":"homeautomation","reachable":true},"swupdate":{"state":"notupdatable","lastinstall":"1980-01-06T03:21:42"},"type":"Extended color light","name":"Lowboard","modelid":"Flex RGBW","manufacturername":"OSRAM","productname":"Extended color light","capabilities":{"certified":false,"control":{"colorgamuttype":"other","ct":{"min":151,"max":370}},"streaming":{"renderer":false,"proxy":false}},"config":{"archetype":"classicbulb","function":"mixed","direction":"omnidirectional"},"uniqueid":"84:18:26:00:00:ca:a7:d0-03","swversion":"V1.03.07"}[39m
[31mTypeError: Cannot read properties of undefined (reading 'swversion')[39m
[31m    at Object.fix (file:///usr/local/lib/node_modules/homebridge-hue/lib/HueLight.js:186:20)[39m
[31m    at HueLight.setConfig (file:///usr/local/lib/node_modules/homebridge-hue/lib/HueLight.js:512:20)[39m
[31m    at new HueLight (file:///usr/local/lib/node_modules/homebridge-hue/lib/HueLight.js:302:10)[39m
[31m    at HueAccessory.exposeLightResource (file:///usr/local/lib/node_modules/homebridge-hue/lib/HueAccessory.js:223:21)[39m
[31m    at HueAccessory.exposeLightResources (file:///usr/local/lib/node_modules/homebridge-hue/lib/HueAccessory.js:213:14)[39m
[31m    at HueAccessory.expose (file:///usr/local/lib/node_modules/homebridge-hue/lib/HueAccessory.js:103:10)[39m
[31m    at HueBridge.exposeResources (file:///usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:571:51)[39m
[31m    at HueBridge.accessories (file:///usr/local/lib/node_modules/homebridge-hue/lib/HueBridge.js:101:18)[39m
[31m    at processTicksAndRejections (node:internal/process/task_queues:95:5)[39m
[31m    at async Promise.all (index 0)[39m
[37m[10/22/2024, 12:20:52 AM] [39m[36m[Hue][39m Hue Bridge: 11 accessories
[37m[10/22/2024, 12:20:52 AM] [39m[36m[Hue][39m masked debug info dumped to /Users/anarqist/.homebridge/homebridge-hue.json.gz
[37m[10/22/2024, 12:20:52 AM] [39mInitializing platform accessory 'Hue Bridge'...
[37m[10/22/2024, 12:20:52 AM] [39m[36m[Hue][39m Hue Bridge: 3 services
[37m[10/22/2024, 12:20:52 AM] [39mInitializing platform accessory 'Livingroom Ceiling Lamp'...
[37m[10/22/2024, 12:20:52 AM] [39mInitializing platform accessory 'Desk Lightstrip'...
[37m[10/22/2024, 12:20:52 AM] [39mInitializing platform accessory 'Bed'...
[37m[10/22/2024, 12:20:52 AM] [39mInitializing platform accessory 'Kitchen Ceiling Lamp'...
[37m[10/22/2024, 12:20:52 AM] [39mInitializing platform accessory 'Hall Ceiling Lamp'...
[37m[10/22/2024, 12:20:52 AM] [39mInitializing platform accessory 'Lowboard'...
[37m[10/22/2024, 12:20:52 AM] [39m[36m[Hue][39m Platform Hue returned an accessory at index 6 with an empty set of services. Won't adding it to the bridge!
[37m[10/22/2024, 12:20:52 AM] [39mInitializing platform accessory 'Lowboard'...
[37m[10/22/2024, 12:20:52 AM] [39m[36m[Hue][39m Platform Hue returned an accessory at index 7 with an empty set of services. Won't adding it to the bridge!
[37m[10/22/2024, 12:20:52 AM] [39mInitializing platform accessory 'Hue motion sensor 1'...
[37m[10/22/2024, 12:20:52 AM] [39mInitializing platform accessory 'Hue motion sensor 3'...
[37m[10/22/2024, 12:20:52 AM] [39mInitializing platform accessory 'Hue dimmer switch 1'...
[37m[10/22/2024, 12:20:52 AM] [39m[32mHomebridge v1.8.4 (HAP v0.12.2) (Hue) is running on port 50482.[39m
```

#### Debug Files
[homebridge-hue.json.gz](https://github.com/user-attachments/files/17474484/homebridge-hue.json.gz)

**Comment by ebaauw** [comment]
Could you try beta v0.14.1-0?

**Comment by anarqist** [comment]
> Could you try beta v0.14.1-0?

Works fine now! Everything is properly exposed, nothing got removed and everything is accessible. The message from the open [issue](https://github.com/ebaauw/homebridge-hue/issues/1190) still remains but if I understand it correctly it is not really an issue at all! 

thank you for the quick fix!

**Comment by ebaauw** [comment]
Released v0.14.1.
